src/Esprima/Utils/ToJavascriptConverter.cs

```diff
@@ -0,0 +1,1410 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Esprima.Ast;
+
+namespace Esprima.Utils
+{
+    public class ToJavascriptConverter
+    {
+        public static string ToJavascript(Node node)
+        {
+            var visitor = new ToJavascriptConverter();
+            visitor.Visit(node);
+            return visitor.ToString();
+        }
+
+        protected StringBuilder _sb = new StringBuilder();
+        protected int _indentionLevel = 0;

```

Comment:
These should be in a `FormattingSettings` class.

Reply:
I've removed this completely, cause it does not work at the moment, so this is something we could do as an extension of it